src/Exceptions/CouldNotSendNotification.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace NotificationChannels\Signal\Exceptions;
+
+class CouldNotSendNotification extends \Exception
+{
+    public static function serviceRespondedWithAnError($response)
+    {
+        return new static("Unable to send message. Please ensure recipient is registered with Signal. Symfony output was: `{$response}['$symfonyerror']`");
```

Comment:
where does `symfonyerror` come from?

Reply:
Could we use `sprintf` please

Reply:
Changed to `sprintf` and `symfonyerror` to `result`.